### ISSUE TITLE
Supporting media_player base join / unjoin for groups (and so support Roon players)

### ISSUE THREAD

**Issue by pavoni** [kalkih/mini-media-player#519]
Not an issue - just an enhancement suggestion.

I support the HA roon integration, and just rewrote the roon specific `join/unjoin` services to use the new base services.

So would be great to be able to take advantage of your group code for roon - and if the player supported the new base services hat should work for other integrations over time,

https://www.home-assistant.io/integrations/media_player/#service-media_playerjoin

**Comment by micha91** [comment]
As I am also working on an official integration, I am using these new base services, too. I have created a small PR, which should add support for these services, without breaking existing configurations.

**Comment by hcoohb** [comment]
It would be great indeed !

**Comment by jodur** [comment]
Yes would be great indeed if new bases services would be supported

**Comment by pavoni** [comment]
Thanks very much for adding this. Works nicely with Roon!
